### DISCUSSION TITLE
直接部署也是报错404.。。。任何参数没有修改

### DISCUSSION THREAD

**Discussion by lanshatop** [spencerwooo/onedrive-vercel-index#804] [General]
![image](https://user-images.githubusercontent.com/117066976/204542109-3b693b10-c8bd-4de2-a089-493c601831b7.png)
![image](https://user-images.githubusercontent.com/117066976/204542177-f0242c9a-f8c8-46ed-95d2-6095888c0837.png)

**Comment by lightumcc** [comment]
Redis环境变量链接配置错误，请检查SSL是否开启，若开启环境变量中应该是rediss://
